app/views/case_logs/index.html.erb

```diff
@@ -1,8 +1,21 @@
+<% item_label = @pagy.count > 1 ? "logs" : "log" %>
+
+<% if @searched.present? %>
```

Comment:
I don't think this will do anything unless you pass it to `content_for :title`?

Reply:
Indeed and we use it on line 9 I think

Reply:
We use item_label in log_list

Reply:
Gotcha